packages/core/src/memdb.ts

```diff
@@ -73,20 +71,20 @@ abstract class MemDb extends TxProcessor implements Storage {
     let result: Doc[]
     if (query._id !== undefined) {
       const obj = this.objectById.get(query._id)
-      result = obj ? [obj] : []
+      result = (obj != null) ? [obj] : []
```

Comment:
`obj !== undefined` -- `get` returns `undefined` if there is no key

Reply:
Good catch, it was auto formatted.